server/views/pages/showIncidents.njk

```diff
@@ -0,0 +1,47 @@
+{% extends "../partials/layout.njk" %}
+{% set pageTitle = applicationName + " - Home" %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% block content %}
+  {% set rows = [] %}
+  {% for inc in incidents %}
+    {% set rows = (rows.push(
+      [
+      { html: "<a href='/incident/" + inc.id + "/'>" + inc.title + "</a>" },
```

Comment:
when generating html, it's always important to escape variables – title might contain special characters. if we're being super-carefuly, id could be escaped but it's safe for now

you could do something like this:
```nunjucks
{% set title %}{{ title }}{% endset %}
```
and 
```javascript
{ html: `<a href="/incident/${inc.id}">${title}</a>` }
```

Reply:
think I have this in place now and seems to work...